pyproject.toml

```diff
@@ -16,7 +16,3 @@ exclude = '''
     | __pycache__
 )/
 '''
-
-[tool.pytest.ini_options]
```

Comment:
This config was disabling `pytest-asyncio` plugin and a lot of async tests were skipped.